src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.service;
 
+// import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
-     * Imagine that some user wants to login to your site.
+     * Imagine that some user wants to log into your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
+     * param email - user's email
+     * param password - user's password
+     * return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+        return email.equals(userService.findByEmail(email).getEmail())
+               && password.equals(userService.findByEmail(email).getPassword());
```

Comment:
You are calling method `userService.findByEmail(email)` three times, better call this method only once